R2API/ItemDrop/Catalog.cs

```diff
@@ -11,63 +11,64 @@ public static class Catalog {
                 These are purely for reference so they can be iterated through elsewhere.
             */
 
-            public static bool Loaded;
```

Comment:
This is needed by the SubmoduleAPI and loading IIRC

Reply:
This class isn't a submodule. It is a utility class for ItemDropAPI and MonsterItemsAPI. So this should be ok.